src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -50,7 +60,10 @@ public function __construct(bool|callable $debug = false, string $charset = null
     {
         $this->debug = \is_bool($debug) ? $debug : $debug(...);
         $this->charset = $charset ?: (ini_get('default_charset') ?: 'UTF-8');
-        $this->fileLinkFormat = $fileLinkFormat ?: (ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format'));
+        if (null === $fileLinkFormat) {
+            $fileLinkFormat = $_SERVER['SYMFONY_IDE'] ?? null;
```

Comment:
`$fileLinkFormat ??= $_SERVER['SYMFONY_IDE'] ?? null;`

Reply:
Thanks for the tip.